src/screens/SurveyDetail/detail.js

```diff
@@ -3,17 +3,28 @@ import React, { useContext } from 'react';
 import { useHistory } from 'react-router-dom';
 
 import { DetailsContext } from 'contexts/details';
+import { SurveyStatusContext } from 'contexts/surveyStatus';
 
 const Detail = () => {
   const detailsContext = useContext(DetailsContext);
+  const { dispatch } = useContext(SurveyStatusContext);
   const history = useHistory();
   const goBack = () => history.goBack();
 
+  const startSurvey = () => {
+    dispatch({
+      type: 'START',
```

Comment:
How about adding a file to keep all survey action types in `/constants/survey.js` as mentioned in [Compass](https://nimblehq.co/compass/development/code-conventions/react/#project-structure)

```
// constants/survey.js
SURVEY_START = 'SURVEY_START';
```

then use it here
```
import * from 'constants/survey'
// ...
dispatch({
      type: SURVEY_START,
});
```